### ISSUE TITLE
SelectOneRadio: selected option not unselectable by keyboard

### ISSUE THREAD

**Issue by jepsar** [primefaces/primefaces#8377]
You can select using space, but if the component has `unselectable="true"` set, you cannot unselected the selected option using space.
